client/src/App.tsx

```diff
@@ -1776,6 +1831,11 @@ function App() {
 			);
 
 			if (isSuccessful) {
+				let oldBackgroundType = "";
+				if(background.type){
+					oldBackgroundType = background.type.valueOf();
```

Comment:
valueOf seems a bit strange, any reason why that's necessary?

Reply:
i used valueOf() because it creates a new var instead of simply pointing to the variable.  this helps because later on i check this value in an if statement which would be incorrect if it was a pointer to background.type since its updated with setBackground()

Reply:
This code didnt work without valueOf? Just seems strange because background.type is a string primitive which doesnt need any sort of deep cloning like with objects.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/valueOf

https://stackoverflow.com/a/50840423/13395599